doc/conf.py

```diff
@@ -304,7 +298,7 @@
 # -- Options for intersphinx extension ---------------------------------------
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'python': ('https://docs.python.org/3', None)}
+intersphinx_mapping = {'https://docs.python.org/': None}
```

Comment:
This change was a fix implemented earlier -make sure it is not lost
See https://github.com/oneapi-src/oneTBB/commit/d405245230cec1726c00318b5eb0d6bad14a331c